iocore/net/quic/QUICDebugNames.cc

```diff
@@ -209,41 +209,51 @@ QUICDebugNames::transport_parameter_id(QUICTransportParameterId id)
 }
 
 const char *
-QUICDebugNames::stream_state(const QUICStreamState &state)
+QUICDebugNames::stream_state(const int state)
 {
-  switch (state.get()) {
-  case QUICStreamState::State::_Init:
-    return "INIT";
-  case QUICStreamState::State::Ready:
-    return "READY";
-  case QUICStreamState::State::Send:
-    return "SEND";
-  case QUICStreamState::State::DataSent:
-    return "DATA_SENT";
-  case QUICStreamState::State::DataRecvd:
-    return "DATA_RECVD";
-  case QUICStreamState::State::ResetSent:
-    return "RESET_SENT";
-  case QUICStreamState::State::ResetRecvd:
-    return "RESET_RECVD";
-  case QUICStreamState::State::Recv:
-    return "RECV";
-  case QUICStreamState::State::SizeKnown:
-    return "SIZE_KNOWN";
-  case QUICStreamState::State::DataRead:
-    return "DATA_READ";
-  case QUICStreamState::State::ResetRead:
-    return "RESET_READ";
-  case QUICStreamState::State::Open:
-    return "OPEN";
-  case QUICStreamState::State::HC_L:
-    return "HC_L";
-  case QUICStreamState::State::HC_R:
-    return "HC_R";
-  case QUICStreamState::State::Closed:
-    return "CLOSED";
-  case QUICStreamState::State::_Invalid:
-    return "INVALID";
+  switch (state) {
+  case 0:
```

Comment:
Is there any reason to not use `QUICSendStreamState::Init` ?

Reply:
Ah, `QUICStreamState` is separated to `QUICSendStreamState` and `QUICReceiveStreamState`.
I understand, but this looks dropping advantages of enum class.